### PR TITLE
fixed markdown headers that needed preceding line break

### PR DESCRIPTION
**Pull request by NReilingh**
These headers currently look like this on docs.microsoft.com:

customize the start screen

export the start layout

### CODE REVIEW

File: windows/configuration/customize-and-export-start-layout.md [hunk 1]
```diff
@@ -46,8 +46,8 @@ You can deploy the resulting .xml file to devices using one of the following met
 -   [Mobile device management (MDM)](customize-windows-10-start-screens-by-using-mobile-device-management.md)
 
 <span id="bkmkcustomizestartscreen" />
```
Comment (illfated) [inline windows/configuration/customize-and-export-start-layout.md]:
```suggestion
```

File: windows/configuration/customize-and-export-start-layout.md [hunk 2]
```diff
@@ -46,8 +46,8 @@ You can deploy the resulting .xml file to devices using one of the following met
...
-## Customize the Start screen on your test computer
```
Comment (illfated) [inline windows/configuration/customize-and-export-start-layout.md]:
```suggestion
```

File: windows/configuration/customize-and-export-start-layout.md [hunk 3]
```diff
@@ -58,6 +58,7 @@ To prepare a Start layout for export, you simply customize the Start layout on a
 2.  Create a new user account that you will use to customize the Start layout.
 
 <a href="" id="bmk-customize-start"></a>
```
Comment (illfated) [inline windows/configuration/customize-and-export-start-layout.md]:
```suggestion
```

File: windows/configuration/customize-and-export-start-layout.md [hunk 4]
```diff
@@ -58,6 +58,7 @@ To prepare a Start layout for export, you simply customize the Start layout on a
...
+
```
Comment (illfated) [inline windows/configuration/customize-and-export-start-layout.md]:
```suggestion
```

**Comment by illfated** [comment]
My suggestions above are based on the Pull Requests #4287 and #4290
(you may want to have a look at the PR files for reference).

**Comment by NReilingh** [comment]
@illfated Thanks, I had a hunch those tags were extraneous at this point. I've removed them.

**Comment by illfated** [comment]
Very good. This PR looks ready for final review.
@Dansimp : This is the same type of HTML tag removal as in the previously merged pull requests #4287 and #4290 (for reference).

**Pull request merged**